newsroom/companies/views.py

```diff
@@ -222,9 +230,10 @@ async def approve_company(args: CompanyItemArgs, params: None, request: Request)
     await service.update(original.id, updates)
 
     # Activate the Users of this Company
-    # TODO-ASYNC: Convert to users async service
-    users_service = get_resource_service("users")
-    for user in users_service.get(req=None, lookup={"company": original.id, "is_approved": {"$ne": True}}):
+    users_service = UsersService()
+    company_users = await users_service.search(lookup={"company": str(original.id)})
```

Comment:
And here. Thinking because we have this in a few locations, it might be good to have a common/util function that provides the list of users for a company

Reply:
yep, agree. I'll a separate method to avoid repeating.

Reply:
done.